frontend/src/shared/globalStyles/global.css

```diff
@@ -1,3 +1,16 @@
+.main {
```

Comment:
Do you actually want to defined this here? App.css also has a `.main`. Maybe move this over there since it's not a tag we use often across the codebase (in fact, we probably use it exactly once).

Reply:
fixed